doc/admin/installation/manual_smallscale.rst

```diff
@@ -50,12 +50,17 @@ Database
 --------
 
 Having the database server installed, we still need a database and a database user. We can create these with any kind
-of database managing tool or directly on our database's shell, e.g. for MySQL::
-
-    $ mysql -u root -p
-    mysql> CREATE DATABASE pretix DEFAULT CHARACTER SET utf8mb4 DEFAULT COLLATE utf8mb4_unicode_ci;
-    mysql> GRANT ALL PRIVILEGES ON pretix.* TO pretix@'localhost' IDENTIFIED BY '*********';
-    mysql> FLUSH PRIVILEGES;
+of database managing tool or directly on our database's shell, e.g. for POSTGRESQL:
```

Comment:
This needs to be a double-colon (``::``) at the end of the line such that it is recognized as a code snippet.